packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -118,8 +130,7 @@ export function ListViewItem(props) {
             }
           )
         }
-        ref={cellRef}
-        {...mergedProps}>
+        role="gridcell">
```

Comment:
We'll probably eventually want to move all this logic into a `useListItem` hook since it's no longer a pure grid. But doesn't have to be in this pr.

Reply:
Ah, good idea, I'll handle in a followup PR once we settle on names and stuff